tests/test_structures.py

```diff
@@ -0,0 +1,75 @@
+# coding: utf-8
+import pytest
+
+from requests.structures import CaseInsensitiveDict, LookupDict
+
+
+class TestCaseInsensitiveDict:
+
+    @pytest.fixture
+    def dict_instance(self):
```

Comment:
This doesn't use the `self` parameter and as such shouldn't be defined on the class. There should also be a docstring here explaining what dict_instance does for a quick glance.

In fact nothing in this class uses `self` and as such each should be its own function.